include/zephyr/dt-bindings/flash_controller/npcx_fiu_qspi.h

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_NPCK_FIU_QSPI_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_NPCK_FIU_QSPI_H_
+
+/**
+ * @name NPCX QSPI definition for configuring the Quad-SPI peripherals
+ * For example:
+ *
+ * int_flash: w25q40@0 {
+ *      compatible ="nuvoton,npcx-fiu-nor";
+ *          status = "okay"
+ *          reg = <0>;
+ *          size = <0x400000>;
+ *          qspi-flags = <(NPCX_QSPI_SW_CS0 | NPCX_QSPI_SEC_FLASH_SL)>;
+ *  };
+ */
+
+/* Software controlled Chip-Select number for UMA transactions */
+#define NPCX_QSPI_SW_CS0	(1 << 0)
```

Comment:
How about using `BIT()` macros around here? Up to you...

Reply:
BIT is more readable. I will update it later.

Reply:
Done.